### ISSUE TITLE
Implement new webdav setup

### ISSUE THREAD

**Issue by hiromipaw** [OpenArchive/Save-app-ios#236]
The new webdav setup is divided in 3 steps:

![Private Server - Log In](https://github.com/OpenArchive/Save-app-android/assets/1177650/1deb4290-bce7-41bb-87b4-b4e026b6fc8c)

![Private Server - Log In-1](https://github.com/OpenArchive/Save-app-android/assets/1177650/8e1a6e9d-d1b9-4e8f-a215-40cf7055fcc1)

![Private Server - Success](https://github.com/OpenArchive/Save-app-android/assets/1177650/9cdf6b0c-04f3-4edf-8cb1-aa3017b76210)

Typography for title and description:
```
font-family: Montserrat;
font-size: 14px;
font-style: italic;
font-weight: 700;
line-height: 16px;
letter-spacing: 0.01em;
text-align: left;

font-family: Montserrat;
font-size: 14px;
font-weight: 500;
line-height: 16px;
letter-spacing: 0.01em;
text-align: left;
```

Form typography:
```
/* Server Info */

width: 331px;
height: 16px;

font-family: 'Montserrat';
font-style: normal;
font-weight: 600;
font-size: 14px;
line-height: 16px;
/* identical to box height, or 114% */
letter-spacing: 0.01em;

/* White */
color: #FFFFFF;


/* Inside auto layout */
flex: none;
order: 0;
align-self: stretch;
flex-grow: 0;

/* ↪ Label */

width: 291px;
height: 24px;

font-family: 'Montserrat';
font-style: normal;
font-weight: 400;
font-size: 13px;
line-height: 18px;
/* or 138% */
display: flex;
align-items: center;

/* White */
color: #FFFFFF;
```

Figma:
https://www.figma.com/file/MngzLdtPsEmvEsUPKsbmE8/Save-App-2.0

Image:
![Group 431](https://github.com/OpenArchive/Save-app-android/assets/1177650/87a13a20-ebf9-48a8-bec2-2b33ef689af9)

**Comment by tladesignz** [comment]
@purvi-ranawat:  This will be contained in version 3.0.0, build 84.

**Comment by tladesignz** [comment]
Ahaha! Got creative with the space naming. Ok, I improved the method to pretty-print a space name.


Fix contained in version 3.0.1, build 88.

**Comment by purvi-ranawat** [comment]
@tladesignz This is working fine now. Tested on iPhone XR, iOS 17.2. version 3.0.1, build 89
